lib/elements/elements.dart

```diff
@@ -125,6 +125,7 @@ class DSplitter extends DElement {
       _offset = e.offset;
 
       _moveSub = document.onMouseMove.listen((e) {
+        querySelector("#frame").style.pointerEvents = "none";
```

Comment:
Right now this library doesn't know anything specific about the application that is using it. What if we added two optional params into the ctor of this class, something like `onDragStart` and `onDragEnd`. If non-null, these would be called by the splitter when dragging started and finished. This would allow a consumer of the splitter to enable and disable pointer events on the iframe, without the splitter itself having to know about the iframe.